packages/oscal-viewer/src/themes/AppTheme.js

```diff
@@ -0,0 +1,37 @@
+import { createTheme } from "@mui/material/styles";
+
+export const appTheme = createTheme({
+  palette: {
+    primary: {
+      main: "#001131",
+    },
+    secondary: {
+      main: "#00BDE3",
+    },
+  },
+  typography: {
+    h1: {
+      fontWeight: "700",
+      fontSize: "50px",
+    },
+    fontFamily: ["Source Sans Pro", "sans-serif"].join(","),
```

Comment:
Can we do the same thing we do for `oscal.io` and use the Roboto font? I started to do that in #809. I am fine with using that same import from Google Fonts.

Generally though, please try to align things with
 - https://github.com/EasyDynamics/oscal.io/blob/main/assets/css/main.scss
 - https://github.com/EasyDynamics/oscal.io/blob/main/_sass/customize.scss
 - https://github.com/EasyDynamics/oscal.io/blob/main/_sass/colors.scss
 - https://github.com/EasyDynamics/oscal.io/blob/main/_sass/base.scss

Reply:
The choice to use Source Sans Pro here was to match Emily's style guide. We may want to have a larger discussion on this as a team before we decide on making this change. I've been hearing some opinions on keeping the tools completely separate including styling - as `oscal.io` is made for the community and will be separate from our own branding, while this tool will fit our own branding.